cmd/eksctl-anywhere/cmd/installpackagecontroller.go

```diff
@@ -72,7 +72,7 @@ func installPackageController(ctx context.Context) error {
 	}
 
 	curatedpackages.PrintLicense()
-	err = ctrlClient.EnableCuratedPackages(ctx)
+	err = ctrlClient.Enable(ctx)
```

Comment:
Any reason for the name change?

Reply:
The old name was super long, and it didn't feel useful to have packagesControllerClient.EnableCuratedPackages, as per

> For instance, the buffered reader type in the bufio package is called Reader, not BufReader, because users see it as bufio.Reader, which is a clear, concise name. 

https://go.dev/doc/effective_go#names